examples/basic-webpack/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Basic CommonJS Example with Webpack</title>
+    <link rel="stylesheet" href="./../shared/css/base.css" />
+  </head>
+  <body>
+    <h1>Basic React Example with Webpack</h1>
+    <div id="container">
+      <p>
+        To install React, follow the instructions on
+        <a href="https://github.com/facebook/react/">GitHub</a>.
+      </p>
+      <p>
+        If you can see this, React is <strong>not</strong> working right.
+        If you checked out the source from GitHub make sure to run <code>grunt</code>.
+      </p>
+    </div>
+    <h4>Example Details</h4>
+    <p>This is basic example how to use React with webpack.
+      Navigate to the example directory
+      <pre>cd /path/to/react/examples/example-name</pre>
+      and run
+      <pre>npm start</pre>
+      <br>
+      for development server run 
+      <pre>npm run dev-server</pre>
```

Comment:
This and the readme should be updated as well to reflect the changes to scripts. Sorry I didn't call that out before as well (thanks for the update, btw).